modules/helm/options.go

```diff
@@ -13,4 +13,5 @@ type Options struct {
 	HomePath       string              // The path to the helm home to use when calling out to helm. Empty string means use default ($HOME/.helm).
 	EnvVars        map[string]string   // Environment variables to set when running helm
 	Version        string              // Version of chart
+	ApiVersions    []string            // List of api-versions to use with the helm template command
```

Comment:
`helm.Options` is intended to capture common options across all the helm commands and I am slightly disinclined to have the precedence of packing it with command specific args. Is it possible to instead have `RenderTemplate` take in variadic string arguments at the end of the function to pass to the command as extra flags? 

E.g., here is what I had in mind:

```go
helm.RenderTemplate(
    // Existing args
    t, options, "chart", "release", []string{},
    // Additional custom args to pass to the template command. This can be anything that template supports, not just --api-versions
    "--api-versions", "helm.k8s.io/test", "--skip-crds",
)
```

Reply:
Makes sense.  My only question is do you want want to use variadic string arguments or a simple array of extra args for RenderTemplate?  Here is my quick mockup.

```
func RenderTemplate(t testing.TestingT, options *Options, chartDir string, releaseName string, templateFiles []string, extraArgs ...string) string {
...
}
```

And then in RenderTemplateE  join the array?
`args = append(args, extraArgs...)`

I can update the PR shortly after I do a test.

Reply:
Yup how you had it is what I had in mind, so that we maintain backwards compatibility.